pilot/pkg/proxy/envoy/v2/mem.go

```diff
@@ -324,15 +324,7 @@ func (sd *MemServiceDiscovery) GetProxyServiceInstances(node *model.Proxy) ([]*m
 func (sd *MemServiceDiscovery) ManagementPorts(addr string) model.PortList {
 	sd.mutex.Lock()
 	defer sd.mutex.Unlock()
-	return model.PortList{{
-		Name:     "http",
-		Port:     3333,
-		Protocol: model.ProtocolHTTP,
-	}, {
-		Name:     "custom",
-		Port:     9999,
-		Protocol: model.ProtocolTCP,
-	}}
+	return nil
```

Comment:
There is no need for locking the mutex anymore

Reply:
I know - there was no need for it before either, and we're not longer using mgmt ports ( due to capture mode). However in 1.1.x we're planning to fix capture, and then we'll need to handle mgmt ports again. I wouldn't go trough another test cycle just to remove the mutex.